src/api/CompileTools.ts

```diff
@@ -557,10 +557,19 @@ export namespace CompileTools {
         getDefaultVariables(instance, ileSetup)
       );
 
-      if (commandString.startsWith(`?`)) {
-        commandString = await vscode.window.showInputBox({ prompt: `Run Command`, value: commandString.substring(1) }) || '';
-      } else {
-        commandString = await showCustomInputs(`Run Command`, commandString, title);
+      if (commandString) {
+        const commands = commandString.split(`\n`).filter(command => command.trim().length > 0);
+        let promptedCommands = [];
```

Comment:
Make this explicitely immutable
```suggestion
        const promptedCommands = [];
```